mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -782,9 +782,10 @@ func ResourceContainerCluster() *schema.Resource {
 				Optional:      true,
 				Computed:      true,
 				ForceNew:      true,
+				Deprecated:    "Deprecated in favor of ip_allocation_policy.cluster_ipv4_cidr_block",
```

Comment:
I don't think that this deprecation is safe, as `ip_allocation_policy.cluster_ipv4_cidr_block` only works when `use_ip_aliases` is `true` (aka the cluster is VPC-native), while this works for routes-based clusters too. The nested version of the field is deprecated in the API, but I think we removed it already clientside.